File: packages/core/tests/fixtures/unhealthy/Complex.cs
public class Complex {
    public string ProcessData(
        object[] items, object config, object options,
        Action callback, string mode, object extra
    ) {
        if (items != null) {
            foreach (var item in items) {
                int val = (int)item;
                if (val > 0) {
                    while (val > 1) {
                        if (val % 2 == 0) {
                            if (mode == "fast") {
                                return "fast";
                            } else if (mode == "slow") {
                                return "slow";
                            }
                        }
                        val--;
                    }
                } else if (val < -10) {
                    return "negative";
                }
            }
        } else if (extra != null) {
            return "extra";
        }
        return "default";
    }
}

File: docs/ochs/conformance/unhealthy/complex.py
def process_data(items, config, options, callback, mode, extra):
    if items:
        for item in items:
            if item > 0:
                while item > 1:
                    if item % 2 == 0:
                        if mode == 'fast':
                            return 'fast'
                        elif mode == 'slow':
                            return 'slow'
                    item -= 1
            elif item < -10:
                return 'negative'
    elif extra:
        return 'extra'
    return 'default'
